students/hyerim/users/views.py

```diff
@@ -1,3 +1,33 @@
-from django.shortcuts import render
+import json
+import re
+
+from django.http import JsonResponse
+from django.views import View
+from users.models import User
```

Comment:
```suggestion
import json
import re

from django.http import JsonResponse
from django.views import View

from users.models import User
```
django가 제공해주는 모듈과 직접 생성한 모듈은 띄어서 구분해주세요!